gnat2goto/driver/tree_walk.adb

```diff
@@ -993,6 +997,21 @@ package body Tree_Walk is
       return Ret;
    end Do_Constant;
 
+   ----------------------
+   -- Do_Real_Constant --
+   ----------------------
+
+   function Do_Real_Constant (N : Node_Id) return Irep is
+      Ret           : constant Irep := New_Irep (I_Constant_Expr);
+      Real_Constant_Type : constant Irep := Do_Type_Reference (Etype (N));
+   begin
+      Set_Source_Location (Ret, Sloc (N));
+      Set_Type (Ret, Real_Constant_Type);
+      --  ??? FIXME
+      Set_Value (Ret, "0");
```

Comment:
We've already noted this down, but again for posterity: This is obviously not the correct value, we're missing an actual floating point implementation at the moment.